code-of-conduct-JP.md

```diff
@@ -0,0 +1,40 @@
+## 行動規範
+
+我々は事業全般を通じ、誰もがハラスメント行為を受けることのない職場環境を作り、いかなる形のハラスメント、尚且つそれの参加者を容認することをしない。
+我々の行動規範として、社員一人一人が他者を尊重し、プロとしての意識をもって行動することを必要とする。
+この規範は社外での活動やその他全面―オンライン、現実を問わず―かつ一対一でのコミュニケーションの際におこる不適切な行動においても適応され、その行動がこのコミュニティーの一員の安全に関わるどの場に置いても適応される。
+
+出展者、演説者、スポンサー、スタッフなどのDocker, Incのイベントの参加者、また、Docker, Incの施設内で行われる行事の参加者はこの行動規範の対象となる。
+Dockerコミュニティは多様性を理解し、多種多様な人材の受け入れを推進している。我々は多様多彩な経歴の持ち主の導入を可能な限り支持し、我々の立場を明確にするよう努力している。
```

Comment:
Translation for "Diversity and inclusion make the Docker community strong." seems missing